test/infrastructure/docker/config/manager/manager.yaml

```diff
@@ -30,16 +30,10 @@ spec:
         volumeMounts:
           - mountPath: /var/run/docker.sock
             name: dockersock
-          - mountPath: /var/lib/docker
-            name: dockerlib
         securityContext:
           privileged: true
       terminationGracePeriodSeconds: 10
       volumes:
         - name: dockersock
           hostPath:
             path: /var/run/docker.sock
-            type: Socket
```

Comment:
Should `type: Socket` be preserved here? Otherwise, lgtm

Reply:
No, that caused some errors. It's best to let Kubernetes figure out what type it should be